collections/_article/godotcon-2025-xr-android-recap.md

```diff
@@ -0,0 +1,135 @@
+
+# Hybrid Apps
+
+Introduced in late 2023, Hybrid Apps are apps that can run both in **Panel** mode or in **Immersive** mode, and are able to **transition between both modes at runtime**. In Panel mode, the app can run side-by-side with other apps, allowing users to multitask. While in Immersive mode, the app’s experience is the sole focus and the user is fully immersed in it.
+
+The feature provides developers with several benefits:
+
+* Developers can easily add immersive support to existing apps and games
+* Developers can leverage existing codebase and logic  
+* Users can select the interaction mode they’re most comfortable with
+
+The Godot XR editor is itself a Hybrid App and has been able to leverage this feature to provide a unique development experience, allowing developers to transition back and forth between the editor window and their immersive projects.   
+With the release of [version 4 of the Godot OpenXR Vendors plugin](https://github.com/GodotVR/godot_openxr_vendors/releases/tag/4.0.0-stable), we’re bringing the same capabilities to all developers in Godot 4.4\!
+
+ <video autoplay loop muted playsinline title="Kenney 3D platformer Hybrid App sample">
+  <source src="/storage/blog/godotcon-2025-android-xr/godot-hybrid-app-kenney-3d-platformer.mp4?1" type="video/mp4">
+</video>
+
+By leveraging **v4** of the Godot OpenXR Vendors plugin, developers can build Hybrid Apps straight from the Godot editor. The feature supports all HorizonOS (Quest) devices, with support for Pico devices coming soon\!
+
+The feature dynamically enables or disables itself based on the target platform. For example, **the same Godot project can run as a regular (panel) app when exported to regular Android devices. And it can run as a full Hybrid App when exported to standalone XR devices**.  
+This allows developers to target multiple audiences, multiple platforms (Android, HorizonOS, PicoOS), and multiple stores (Google Play store, Meta Horizon store, Pico store) with the same project.
+
+To learn more about the feature, please check out [the documentation](https://godotvr.github.io/godot_openxr_vendors/manual/hybrid_apps.html).
+
+# Integrating with Android APIs
+
+The Android platform has numerous APIs as well as a rich ecosystem of third-party libraries with wide and diverse functionality. Godot has long provided an [Android plugin system](https://docs.godotengine.org/en/stable/tutorials/platform/android/android_plugin.html) which enables developers to create a Godot interface in order to access and use Android APIs or third-party libraries in their projects.  
+Writing an Android plugin however requires knowledge of Java or Kotlin code, which most Godot developers do not have. As such there are many Android APIs and third-party libraries that don’t have a Godot plugin that Godot developers can interface with. In fact, this is one of the main reasons that developers cite for not being able to switch to Godot from other game engines.
```

Comment:
```suggestion

Writing an Android plugin however requires knowledge of Java or Kotlin code, which most Godot developers do not have. As such there are many Android APIs and third-party libraries that don’t have a Godot plugin that Godot developers can interface with. In fact, this is one of the main reasons that developers cite for not being able to switch to Godot from other game engines.
```
Making this its own paragraph instead of a simple line break